views/components/Sidebar/index.jsx

```diff
@@ -13,10 +13,7 @@ const Sidebar = ({ user, sidebarOpen, onLogout }) => {
       PaperProps={{
         sx: {
           padding: '8px',
-          position: 'fixed',
-          left: 0,
-          top: 0,
-          bottom: 0,
+          position: 'static',
```

Comment:
we don't want this to be statically positioned. We want it fixed so that when the user scrolls it stays fixed.

Reply:
gotcha it's put back
[a8fa89f](https://github.com/AMP-SCZ/dpdash/pull/478/commits/a8fa89f3d10b6e4db122a9aa52573c428d3682dc)